libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-details.component.ts

```diff
@@ -190,10 +198,16 @@ import { MeteringPointDetails } from './types';
 export class DhMeteringPointDetailsComponent {
   modalService = inject(WattModalService);
 
-  meteringPoint = input.required<MeteringPointDetails | undefined>();
+  meteringPointDetails = input.required<MeteringPointDetails | undefined>();
+
+  meteringPoint = computed(() => this.meteringPointDetails()?.currentMeteringPointPeriod);
+
+  commercialRelation = computed(() => this.meteringPointDetails()?.currentCommercialRelation);
+
+  instationAddress = computed(() => this.meteringPoint()?.installationAddress);
```

Comment:
nitpick: spelling. Should be `installationAddress`

Reply:
Thx